composer.json

```diff
@@ -19,15 +19,15 @@
         "php": ">=7.3.0,<7.5.0",
         "doctrine/common": "^2.6",
```

Comment:
```
"doctrine/common": "^2.6 || ^3.0",
```

Reply:
Umm this is intersting. Why? Does it also applies to `dbal`? I mean for the consistency, do we need to provide more relaxed library versions? If the answer is yes, what would be other good candidates?

Reply:
I don't mind bumping dependency ranges and restricting.

A good strategy to ease upgrading is relaxing in one minor, then bumping in another minor

Reply:
Sorry, it's my fault. I thought it was a PR for upgrade to ORM `2.8`.